redaxo/src/core/pages/mediapool.media.list.php

```diff
@@ -228,7 +228,7 @@
 if ('' != $mediaName) {
     $filter['term'] = $mediaName;
 
-    if ('global' != rex_addon::get('mediapool')->getConfig('searchmode', 'local') && 0 != $rexFileCategory) {
+    if ('global' != rex::getConfig('searchmode', 'local') && 0 != $rexFileCategory) {
```

Comment:
würde ich spezifizieren:

```suggestion
    if ('global' != rex::getConfig('mediapool_searchmode', 'local') && 0 != $rexFileCategory) {
```

(eventuell schaffe ich diesen search-mode auch noch ganz ab)

Reply:
Würde ich abschaffen und `local` als Standard verwenden.